src/main/scala/is/hail/expr/ir/ExtractAggregators.scala

```diff
@@ -50,7 +50,8 @@ object ExtractAggregators {
 
   private def newAggregator(ir: ApplyAggOp): RegionValueAggregator = ir match {
     case x@ApplyAggOp(a, op, args, typ) =>
-      val constfb = FunctionBuilder.functionBuilder[Region, RegionValueAggregator]
+      val constfb =
+        new EmitFunctionBuilder[AsmFunction1[Region, RegionValueAggregator]](Array(GenericTypeInfo[Region]()), GenericTypeInfo[RegionValueAggregator]())
```

Comment:
can we replicate the convenience interface from `FunctionBuilder` on `EmitFunctionBuilder`? This is quite a bit harder to parse now.

Reply:
And down below the emitFB calls could all become:

```
EmitFunctionBuilder.functionBuilder[...]
```

maybe we should get a better name. `apply`? or `make`?